version/1.0.md

```diff
@@ -8,67 +8,62 @@ current: true
 The Open Science Company Manifesto
 ==================================
 
-This manifesto aims at defining fundamental statements every Company running
-with the Open Science initiative in mind must respect and promote.
+This manifesto aims to define the fundamental statements that all companies
+running with the Open Science initiative in mind must respect and promote.
 
-## Open Science pillars are [the Company] culture
+## Open Science pillars as [the Company]'s culture
 
-Unless [the Company] has good reasons for not respecting them (such as National
-Security), every principle of the Open Science initiative should be considered
-as an organization's *credo*.
+Unless [the Company] has reasonable reasons (e.g., National Security concerns),
+every principle of the Open Science initiative should be treated as
+[the Company]'s *credo*.
 
-Each time [the Company] produces source code that could help other scientists in
-their everyday work (software, data analysis, web servers, etc.), the code
-should be open and stored in a public forge that eases community contributions
-and explicitly states on its licensing and credits.
+Each time [the Company] produces source code that may help other scientists in
+their everyday work (e.g., software, data analysis, web servers), the code
+should be made publicly available, allow for community contributions, and be
+clearly accompanied by any licensing information and/or credits.
 
-If relevant for the scientific community, scientific data collected by [the
-Company] should be stored in a public repository that explicitly states on its
-licensing and credits. [The Company] should carefully pay attention to the
-repository terms & conditions before submitting data to this repository.
-Institutional repositories (such as [Zenodo](https://zenodo.org)) must be the
-primary choice, but, third-party repositories can also be considered if the
-service maintainer (company or institution) explicitly states on the future of
-stored data when that maintainer status evolves (fusion, acquisition,
-liquidation).
+Any data collected by [the Company] that is relevant to the scientific community
+should be stored in a public repository, also explicitly accompanied by any
```

Comment:
Maybe add "that fosters community contributions" after "public repository" as suggested by @kazoni here: https://github.com/TailorDev/manifesto/pull/15/commits/e694bcc0ecc5b0391d26b64a7a294e089f98abee